model/distro/distro.go

```diff
@@ -213,6 +219,7 @@ type HostAllocatorSettings struct {
 	Version                string        `bson:"version" json:"version" mapstructure:"version"`
 	MinimumHosts           int           `bson:"minimum_hosts" json:"minimum_hosts" mapstructure:"minimum_hosts"`
 	MaximumHosts           int           `bson:"maximum_hosts" json:"maximum_hosts" mapstructure:"maximum_hosts"`
+	RoundRule              int           `bson:"round_rule" json:"round_rule" mapstructure:"round_rule"`
```

Comment:
Can we call this "RoundingRule" across the board? "RoundRule" is kind of confusing without context. Maybe even "RoundingHostsRule" for the admin settings, because there isn't that context of hostAllocatorSettings.